stdlib/Statistics/src/Statistics.jl

```diff
@@ -752,11 +752,11 @@ julia> median([1 2; 3 4], dims=1)
  2.0  3.0
 ```
 """
-median(v::AbstractArray; dims=:) = _median(v, dims)
+median(v::AbstractArray; dims=:, lt=isless, by=identity, middle=middle, rev=false) = _median(v, dims, lt, by, middle, rev)
 
-_median(v::AbstractArray, dims) = mapslices(median!, v, dims = dims)
+_median(v::AbstractArray, dims, lt, by, middle, rev) = mapslices(median!, v, dims = dims)
```

Comment:
I am not sure on how to handle arguments here. Can someone please help?

Reply:
Pass an anonymous function to `mapslices` perhaps?